.github/pull_request_template.md

```diff
@@ -0,0 +1,26 @@
+### Checklist for PR maker
+- Have you added a changelog? Each package have a changelog.
+- Have you added a test for your PR (if needed)? When adding new feature or discovering a bug that isn't cover by a test, please add a test
```

Comment:
Add a test to this repository, or where? Do we ask for tests for all the changes in the plugins? (I think only a very small percentage of them end up having tests added _here_)